src/components/Footer.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import styled from 'styled-components';
+import {useSelector, useDispatch} from 'react-redux';
+
+import Button from './Button.jsx';
+import Format from './icons/Format.jsx'
+import { formatRequest } from '../store/slices/requestSlice';
+import { valueSelector } from '../store/slices/selectors.js';
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* margin-top: 15px; */
+  padding: 11px;
+  border-top: 1px solid rgba(0, 0, 0, 0.2);
+
+  background-color: #FFFFFF;
+`;
+
+const LinkGithub = styled.a`
+  position: relative;
+  display: block;
+
+  font-size: 16px;
+  line-height: 20px;
+  color: #999999;
+  text-decoration: none;
+
+&:before, &:after {
+  content: '';
+  position: absolute;
+  width: 0%;
+  height: 1px;
+  bottom: 0;
+  margin-top: -0.5px;
+  background: #999999;
+}
+
+&:before {
+  left: -2.5px;
+}
+&:after {
+  right: 2.5px;
+  background: #999999;
+  transition: width 0.8s cubic-bezier(0.22, 0.61, 0.36, 1);
+}
+
+&:focus {
+  outline: none;
+  color: #333333;
+}
+
+&:hover:before {
+  background: #999999;
```

Comment:
If you use color more then 1 time - put it into variable. In case you'll need rebranding and customer would like to change color theme - it would be much easier to switch it there.

Reply:
Pulled colors into separate file

Reply:
LGTM